### DISCUSSION TITLE
async* in es5

### DISCUSSION THREAD

**Discussion by phamconganh** [gildas-lormeau/zip.js#369] [Q&A] [answered]
when I run `zipjs` in `chrome 58`. It not has token `*` in `async* getEntriesGenerator`, then error `Uncaught Syntaxerror: Unexpected token *`. How can I resolve this problem? or zipjs not support es5?

**Comment by gildas-lormeau** [comment, marked as answer]
I just added https://github.com/gildas-lormeau/zip.js/blob/master/rollup-es5.config.js and https://github.com/gildas-lormeau/zip.js/blob/master/rollup-es5.config.js which allows you to build zip.js with rollup and babel and make it compatible with older engines. See the comment in the file for the instructions.

**Reply by rosioc** [reply]
I don't have one
